src/components/ScanOrder.js

```diff
@@ -77,50 +82,35 @@ const ScanOrder = (props) => {
   };
 
   useEffect(() => {
-    retrieveStatuses();
-  }, []);
-
-  const dynamicSort = (property) => {
-    let sortOrder = 1;
-
-    if (property[0] === "-") {
-      sortOrder = -1;
-      property = property.substr(1);
+    if (statuses.length === 0) {
+      retrieveStatuses();
     }
-
-    return function (a, b) {
-      let result =
-        a[property] < b[property] ? -1 : a[property] > b[property] ? 1 : 0;
-      return result * sortOrder;
-    };
-  };
+  }, [statuses]);
 
   let nextDesc = "";
   let nextId = null;
   let nextSeq = null;
   let nextStatusFedOfficeCode = "";
+  let sortedStatuses = [];
 
-  if (STATUSES && order) {
-    STATUSES.sort(dynamicSort("sequence_num"));
+  if (statuses && order) {
+    sortedStatuses = numSort(statuses, "sequence_num", "asc");
```

Comment:
It's probably good to be extra sure, but I'm surprised that the statuses would/could come out of the database unsorted..?

Reply:
they come out of the database sorted by id, which is like the row number on a table.  I'd assume a future coder would write future statuses in a way that kept the sequence_id in order but I don't have a way to enforce that.

This way I'm not dependant on them being written meticulously.